### ISSUE TITLE
Simplestaking app cant send tezos to a wallet

### ISSUE THREAD

**Issue by katalinl** [trezor/trezor-suite#4881]
Simplestaking app cant send tezos to a wallet, it comes up with a warning, no description, and DONE button, on the last step for verification. Although I did verify on the trezor wallet.

**Comment by tsusanka** [comment]
@wendys-cats could you please retest this is indeed the case?

**Comment by wendys-cats** [comment]
Closing issue. **QA OK**
Was unable to reproduce this issue. However, found other two (to be investigated, might be connect related).

I was able to send [transaction ](https://tzstats.com/tz1gvUQbs2iNT8myNgDde6GV7K9FPrXE1Dqt) with TT on simplestaking. 

Device: TT 2.4.3 public regular
Browser: Chrome Version 98.0.4758.102 (Official Build) (64-bit)
Wallet: SimpleStaking 0.7.8
Passphrase: yes



![image](https://user-images.githubusercontent.com/58593111/155983488-15848a34-92dd-439b-9667-9d6f36df1ff4.png)
